static/src/javascripts/projects/commercial/modules/prebid/bid-config.js

```diff
@@ -276,24 +277,37 @@ const getXaxisPlacementId = (sizes: PrebidSize[]): number => {
 /* testing instrument */
 // Returns a map { <bidderName>: true } of bidders
 // according to the pbtest URL parameter
-const pbTestNameMap = memoize(
-    (): {} =>
+
+type TestNameMap = { [string]: boolean };
+
+const pbTestNameMap: () => TestNameMap = memoize(
+    (): TestNameMap =>
         new URLSearchParams(window.location.search)
             .getAll('pbtest')
             .reduce((acc, value) => {
                 acc[value] = true;
                 return acc;
             }, {}),
-    (): String =>
+    (): string =>
```

Comment:
`String` to `string` - generally it is the latter we want since types for literal values are lowercase

https://flow.org/en/docs/types/primitives/

Reply:
👍